spec.bs

```diff
@@ -588,48 +590,77 @@ members:
         automatically.
 </dl>
 
-### Steps to check if a payment can be made ### {#sctn-steps-to-check-if-a-payment-can-be-made}
+### Payment Method additional data type ### {#sctn-payment-method-additional-data-type}
 
-The [=steps to check if a payment can be made=] for this payment method, for an
+The [=payment method additional data type=] for this payment method is
+{{SecurePaymentConfirmationRequest}}.
+
+### Steps to validate payment method data ### {#sctn-steps-to-validate-payment-method-data}
+
+The [=steps to validate payment method data=] for this payment method, for an
 input {{SecurePaymentConfirmationRequest}} |data|, are:
 
 <wpt>
   constructor.https.html
-  authentication-invalid-icon.https.html
-  authentication-icon-data-url.https.html
-  authentication-rejected.https.html
 </wpt>
 
-
 1. If |data|["{{SecurePaymentConfirmationRequest/credentialIds}}"] is empty,
```

Comment:
Did you mean to sync up with this [Chrome code](https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/renderer/modules/payments/secure_payment_confirmation_helper.cc;l=40-101;drc=cfd951a304bb7a1e6424c614c094f5bc581ca8b7)? That code should be executing in the `PaymentRequest()` constructor, as far as I can see. Its sequence of checks is:

1. Credential ID list: non-empty.
2. Timeout: under 1 hour.
3. Instrument display name: non-empty.
4. Instrument icon: valid URL.
5. Either payee origin or name: non-empty.
6. RP ID: non-empty.

This spec here appears to have:

1. Credential ID list: non-empty. ✅
2. Timeout: no checks. (Spec may want to add this.) ⬅️
3. Instrument icon: no checks. (Spec should check this.) 🔥
4. Challenge: non-empty. (Chrome should start doing this.) 🔥
5. RP ID: valid domain. (Chrome should start doing this.) 🔥
6. Either payee name or origin: non-empty. ✅
7. Payee origin: valid https URL. (Chrome should start doing this.) 🔥
8. Instrument display name: non-empty. (Spec may want to move this up to position 3.) ⬅️

There are some differences here, including ordering of checks. Am I looking at the wrong chunk of Chrome code by accident?